backend/backend/settings.py

```diff
@@ -33,13 +33,16 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = config("DJANGO_SECRET_KEY", "dev_not_so_secret_key")
 
-ALLOWED_HOSTS = [
-    HOSTNAME,
-    "localhost",
-    "127.0.0.1",
-    ".osc-fr1.scalingo.io",
-    "staging-ecobalyse.incubateur.net",
-]
+ALLOWED_HOSTS = config(
+    "ALLOWED_HOSTS",
+    [
+        HOSTNAME,
+        "localhost",
+        "127.0.0.1",
+        ".osc-fr1.scalingo.io",
+        "staging-ecobalyse.incubateur.net",
+    ],
+)
```

Comment:
So we should add an example of it in `.env.sample`, and probably set the defaults as just `HOSTNAME` and `127.0.0.1` so the settings module is agnostic to our specific infrastructure requirements and therefore makes the project easier to self-host.

Reply:
Agree. For the defaults I suppose it will require an update of the ENV variables on scalingo, that's why I left it untouched for now.

Reply:
Done here https://github.com/MTES-MCT/ecobalyse/pull/612/commits/8b3ba114acf359d13f97dc960b9d7b05c6414f1b